### PR TITLE
Documentation for 12.5 features

### PR DESCRIPTION
**Pull request by psatyal**
This commit covers documentation for 12.5 features, including:

- Individual non-working days
- Project activity module
- Nextcloud: link and upload files from OpenProject
- The new archive project permission (open to non-admin roles)
- Default work package views (missing docs)

Remaining:

- Screenshots (after feature freeze)
- iCal export (in dev)

### CODE REVIEW

**Comment by psatyal** [comment]
@as-op This is of course on-going, but, as always, could you please check the links? Merci !

File: docs/system-admin-guide/working-days/README.md [hunk 1]
```diff
@@ -17,9 +17,17 @@ To change this setting, navigate to *Administration* → *Working days*.
 
 To change this setting, unselect days that you would like to define as non-working, and select ones that you would like to consider working, and click on **Save**.
 
+### Specific non-working days
+
+You can also designate specific dates (such as national or local holidays) as non-working days. Work packages in regular scheduling mode will skip these days at they will not be included when calculating duration.
```
Comment (marcalcobe) [inline docs/system-admin-guide/working-days/README.md]: @psatyal do we want to emphasise here also that this non-working days work for all the instance and it is not individual?

File: docs/user-guide/activity/README.md [hunk 2]
```diff
@@ -8,39 +8,67 @@ keywords: activity
...
 
-![Activity](1567416672913.png)
+Starting version 12.5, the Activity module also lists changes to project attribuets, including project custom fields. For each update, you will see:
+
+- the name of the project or sub-project
+- the user who was responsible for the change
+- the date and time of the (aggregated) changes
+- a list of attributes that were changed
+
+> IMG: An example of a project attribute update
```
Comment (marcalcobe) [inline docs/user-guide/activity/README.md]: Hi @psatyal,
I suppose we are missing the screenshots here no?

File: docs/user-guide/nextcloud-integration/README.md [hunk 3]
```diff
@@ -61,7 +61,71 @@ The following video gives you a short overview of how to use this integration:
 
 ![OpenProject Nextcloud integration video](https://openproject-docs.s3.eu-central-1.amazonaws.com/videos/OpenProject-Nextcloud-Integration-2.mp4)
 
-Liking files to OpenProject work packages is currently only available via Nextcloud. First, in Nextcloud, navigate to the file or folder that you want to link to a work package and click on the *three dots → **Details**.*
+
+### In OpenProject
+
+In addition to listing files attached to a work package, the **Files** tab now also shows you Nextcloud files that are linked to the current work package. Hovering on any linked file with your mouse will give you options to  **open or download the file, show the containing folder in Nextcloud or remove the link**.
+![Empty_status_files](1_0_00-No_files_linked.png)
```
Comment (marcalcobe) [inline docs/user-guide/nextcloud-integration/README.md:73]: @psatyal this images are old as they miss the function to upload or link existing in Nextcloud.

File: docs/user-guide/nextcloud-integration/README.md [hunk 4]
```diff
@@ -61,7 +61,71 @@ The following video gives you a short overview of how to use this integration:
...
+
+![OP_linked_files](1_1_00-All_files_available.png)
+
+
+To link a Nextcloud file to the current work package, you can either:
+
+- select a file from your computer, which will be uploaded to Nextcloud and linked to this work package
+- select an existing file in Nextcloud to link to
+
+#### Upload and link file(s)
```
Comment (marcalcobe) [inline docs/user-guide/nextcloud-integration/README.md:83]: @psatyal and @Kharonus is this ready? Will the code flag be removed with 12.5?

**Review by marcalcobe — CHANGES_REQUESTED** [review]
Hi @psatyal,

I've added some review comments. The biggest concern I have right now is if the uploading files to Nextcloud from OpenProject will be really released in 12.5 (I tagged @Kharonus for clarification).

File: docs/system-admin-guide/working-days/README.md (hunk 1 above)
Comment (psatyal) [reply docs/system-admin-guide/working-days/README.md]: Hi @marcalcobe. Yes, this is explained in the paragraphe below titled "effect on scheduling":

> As an instance-level setting, any change here will affect the scheduling of all work packages in all projects in that instance. It is currently not possible to define working days at a project-level.

File: docs/user-guide/activity/README.md (hunk 2 above)
Comment (psatyal) [reply docs/user-guide/activity/README.md]: Yep, all screenshots will be added after the feature freeze in one go. I've just left placeholders now. We don't have to include them in this PR; will make another one for the screenshots when all the features are implemented.

File: docs/user-guide/nextcloud-integration/README.md (hunk 3 above)
Comment (psatyal) [reply docs/user-guide/nextcloud-integration/README.md:73]: Noted, thank you! Will include when update screenshots after feature freeze.

**Review by as-op — APPROVED** [review]

**Comment by psatyal** [comment]
Based to @MayaBerd's feedback, added missing screenshots on these pages I edited for 12.5:

- Activities
- Work package views
- Non-working days

And this page I had not edited for 12.5:

- Agile boards

File: docs/user-guide/nextcloud-integration/README.md (hunk 4 above)
Comment (psatyal) [reply docs/user-guide/nextcloud-integration/README.md:83]: Can we get a confirmation that this will be included in 12.5? @Kharonus

**Comment by birthe** [comment]
Updated the section for advanced filters and renamed it to "Filter for attachment file name and content".
@as-op I have changed a few headlines with my commit. Could you please double-check that we do not have any links to these sections? Thanks a lot

**Pull request merged**